File: Java/1168.java
import java.util.Scanner;

public class Main{
    public static void main (String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int ymd = sc.nextInt();
        int umm = sc.nextInt();
        
        if(umm==1 || umm==2) {
        	System.out.println(2012 - 1900 - ymd/10000 + 1);
        } else {
        	System.out.println(2012 - 2000 - ymd/10000 + 1);
        }
    }
}

File: C++/1168.cpp
#include <iostream>

using namespace std;

int main(){
	int ymd, _;
	cin >> ymd >> _;
	if(_==1 || _==2){
		cout << 2012 - 1900 - ymd/10000 + 1;
	} else{
		cout << 2012 - 2000 - ymd/10000 + 1;
	}
}